packages/stacked/example/integration_test/app_test.dart

```diff
@@ -0,0 +1,295 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:new_architecture/main.dart' as app;
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  group('end-to-end test', () {
```

Comment:
I think these names can be more descriptive. I also don't like the fact that it's one large test, BUT, it does get the job done that we want for now so I'm happy with that. Only changes that I'd want is:

Change group name to "Integration Test"
Change test name to "Ensure all stacked integrations still work as expected"

Reply:
😶‍🌫️ Yeah. thanks! I will work on regrouping it to it's specific test. I was just ensuring end to end works for now 😄